### ISSUE TITLE
Do you consider human_thres and Object_thres?

### ISSUE THREAD

**Issue by mansooreh1** [DirtyHarryLYL/DJ-RN#76]
Hello
Do you consider human_thres and Object_thres in test phase such as previous methods?

**Comment by Foruck** [comment]
Yes, and we optimized the thresholds for different object categories considering the frequency in the whole dataset.

**Comment by mansooreh1** [comment]
Thanks. Where did you include this threshold in the code?
